### PR TITLE
Release 0.4.0

### PR DESCRIPTION
**Pull request by shinh**

### CODE REVIEW

**Comment by ukai** [comment]
need to update ChangeLog in top directory?

**Comment by EwoutH** [comment]
At least #409/#421 should be fixed before 0.4.

And it's kind of sad that nothing was done with #156 at the time.

**Comment by sergiud** [comment]
@EwoutH  #156 is incomplete. You're welcome to help finish the PR.

File: src/windows/glog/logging.h [hunk 1]
```diff
@@ -1155,9 +1151,14 @@ class GOOGLE_GLOG_DLL_DECL LogMessage {
   // 2005 if you are deriving from a type in the Standard C++ Library"
   // http://msdn.microsoft.com/en-us/library/3tdb471s(VS.80).aspx
   // Let's just ignore the warning.
-GLOG_MSVC_PUSH_DISABLE_WARNING(4275)
+#ifdef _MSC_VER
+# pragma warning(push)
+# pragma warning(disable: 4275)
+#endif
```
Comment (sergiud) [inline src/windows/glog/logging.h]: This and the next change should be reverted.

**Comment by ukai** [comment]
#421 has been fixed.

how about #409 ?

do we want to merge #156, #425?

**Comment by shinh** [comment]
Re #425: Yeah, I've merged it. Sorry for the latency of the review. There was a comment to ukai-san: https://github.com/google/glog/pull/425#discussion_r253248029

**Comment by shinh** [comment]
I think it'd be better to release 0.4.0 without fixing #156 and #409, considering nothing have been done for them?

**Comment by ukai** [comment]
sgtm.

**Comment by shinh** [comment]
I'm planning to wait: https://github.com/conda-forge/glog-feedstock/issues/14

**Comment by shinh** [comment]
Sorry for no update for a month. Let's get this merged.

**Pull request merged**